pkg/platform/client.go

```diff
@@ -122,6 +122,13 @@ func NewClientFromConfig(ctx context.Context, config *config.CLI) (Client, error
 	return c, nil
 }
 
+func NewClientFromConfig(_ context.Context, config *config.CLI) (Client, error) {
+	c := &client{
+		config: config,
+	}
+	return c, nil
+}
+
```

Comment:
```suggestion
func NewClientFromConfig(config *config.CLI) Client {
	return &client{
		config: config,
	}
}
```

Though, this requires the remaining callers to be adjusted as well :)